.github/workflows/pathogen-repo-build.yaml.in

```diff
@@ -218,6 +218,16 @@ jobs:
           | "$NEXTSTRAIN_GITHUB_DIR"/bin/json-to-envvars
           | tee -a "$GITHUB_ENV"
 
+      - name: Create runtime AWS IAM inline session policy
+        env:
+          FULL_REPO_NAME: ${{ inputs.repo }}
+        run: |
+          export REPO_NAME="${FULL_REPO_NAME#*/}"
+          export PREFIX="${REPO_NAME}_"
+
+          aws_inline_session_policy=$("$NEXTSTRAIN_GITHUB_DIR"/bin/interpolate-env < "$NEXTSTRAIN_GITHUB_DIR"/text-templates/repo-aws-iam-policy.json | jq -c .)
```

Comment:
We could use `jq` to do the templating here in a JSON-aware way instead of the string-based templating of `interpolate-env`. For example:

```bash
jq --from-file "$NEXTSTRAIN_GITHUB_DIR"/text-templates/repo-aws-iam-policy.json --null-input --compact
```

and then in `repo-aws-iam-policy.json` use jq's interpolation, e.g.:

```jq
"s3:prefix": [
  "files/workflows/\(env.REPO_NAME)/*",
  "files/datasets/\(env.REPO_NAME)/*",
  "\(env.PREFIX)*.json"
]
```

I might rename that template from `.json` → `.jq` and/or maybe move it to a new `json-templates/` directory.

Reply:
No longer relevant with [new approach](https://github.com/nextstrain/.github/compare/b2c38a04e15c817dba16b360eefbd2d89ecaaf11..58ba6a0afd97a30bfcf9fee85dbe48e7c9526b40).